### DISCUSSION TITLE
Welcome to zsa-voyager-vscode Discussions!

### DISCUSSION THREAD

**Discussion by gilesknap** [gilesknap/notes#2] [Announcements]
Hey Everyone. Feel free to post here if you want to discuss vscode or ZSA Voyager.

**Comment by kbaikov** [comment]
Hello,

It is worth mentioning that in windows the Hyper shortcut is taken.
You can learn about the all possible combinations here:
https://www.theverge.com/2023/9/15/23874798/microsoft-windows-linkedin-onenote-keyboard-shortcut

So in order to use it you'll have to disable the shortcut in windows registry like described here:
https://answers.microsoft.com/en-us/windows/forum/all/disabling-the-office-hotkey-in-windows-10/58e2f30f-8556-4f73-bf0c-56544697218d

Worked for me :)

**Reply by gilesknap** [reply]
Thanks - did not know that (I've been trying to stay away from  WIndows for the last 18 years - since I left Microsoft).

There is also the Meh key that you can program instead which is ctrl-alt-shift if you did not want to remove Windows hyper. Not sure if there will be some clashes with that?
